.azuredevops/pipelineTemplates/jobs.validateModulePester.yml

```diff
@@ -51,6 +52,7 @@ parameters:
   checkoutRepositories: ''
   # Logic-related parameters
   modulePath: '$(modulePath)'
+  globalModuleTestFilePath: '$(globalModuleTestFilePath)'
```

Comment:
Question: After renaming the file, should we also remove all other references to the `global` test file?

This will primarily affect pipelines. So with a proper `search & replace` this should be a relatively smooth ride.

Reply:
Renamed the variable to `moduleTestFilePath`. Removed related references to `global` from the Wiki and from the Pester test result title.